src/main/java/mate/academy/Main.java

```diff
@@ -1,16 +1,63 @@
 package mate.academy;
 
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import mate.academy.lib.Injector;
+import mate.academy.model.CinemaHall;
 import mate.academy.model.Movie;
+import mate.academy.model.MovieSession;
+import mate.academy.service.CinemaHallService;
 import mate.academy.service.MovieService;
+import mate.academy.service.MovieSessionService;
 
 public class Main {
-    public static void main(String[] args) {
-        MovieService movieService = null;
+    private static final Injector injector = Injector.getInstance("mate.academy");
 
+    public static void main(String[] args) {
+        MovieService movieService = (MovieService) injector.getInstance(MovieService.class);
         Movie fastAndFurious = new Movie("Fast and Furious");
         fastAndFurious.setDescription("An action film about street racing, heists, and spies.");
         movieService.add(fastAndFurious);
         System.out.println(movieService.get(fastAndFurious.getId()));
         movieService.getAll().forEach(System.out::println);
+
+        Movie godfather = new Movie("The GodFather");
+        godfather.setDescription("A cult gangster drama of 1972");
+        movieService.add(godfather);
+        System.out.println(movieService.get(godfather.getId()));
+        movieService.getAll().forEach(System.out::println);
+
+        CinemaHallService cinemaHallService
+                = (CinemaHallService) injector.getInstance(CinemaHallService.class);
+        CinemaHall cinemaHallNumber1 = new CinemaHall();
+        cinemaHallNumber1.setCapacity(12);
+        cinemaHallNumber1.setDescription("Hall 1");
+        cinemaHallService.add(cinemaHallNumber1);
+        System.out.println(cinemaHallService.get(cinemaHallNumber1.getId()));
+
+        CinemaHall cinemaHallNumber2 = new CinemaHall();
+        cinemaHallNumber2.setCapacity(22);
+        cinemaHallNumber2.setDescription("Hall 2");
+        cinemaHallService.add(cinemaHallNumber2);
+        System.out.println(cinemaHallService.getAll());
+
+        MovieSession movieSessionFastAndFurious = new MovieSession();
+        final MovieSessionService movieSessionService
```

Comment:
```suggestion
        final MovieSessionService movieSessionService
```

Move all Service objects to the beginning and make them final